makefile

```diff
@@ -0,0 +1,34 @@
+.PHONY: install-devel test lint pytest check-clean-tree fix force-fix venv
+
+# Customize your venv name by running make as "ZT_VENV=my_venv_name make <command>"
+
+BASEPYTHON?=python3
+ZT_VENV?=zt_venv
```

Comment:
Should we add the default directory to `.gitignore`?

Reply:
Thanks - yes, this was on my to-do but I forgot to add it :+1: